File: CiscoPCCE.Toolkit/Bean/AsrServerXMLList.cs
using System.Xml.Serialization;

using CiscoPCCE.Toolkit.Sdk;

namespace CiscoPCCE.Toolkit.Bean
{
    [XmlRoot("asrServers")]
    public class AsrServerXMLList : BaseApiBean
    {
    [XmlElement("asrServer")]
        public List<AsrServer>? Items { get; set; }
    }
}

File: toolkit/bean/AsrServerXMLList.java
// ----------------------------------------------
package com.cisco.ccbu.cce.unifiedconfig.toolkit.bean;

import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiBean;
import javax.xml.bind.annotation.XmlSeeAlso;
import javax.ws.rs.Path;
import javax.xml.bind.annotation.XmlRootElement;
import java.util.List;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.ReferenceBean;
import java.util.Date;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiListBean;

import javax.xml.bind.annotation.XmlElement;

import javax.xml.bind.annotation.XmlElementWrapper;


@XmlRootElement(name="asrServers")
public class AsrServerXMLList extends BaseApiBean {
  private List<AsrServer> items;

  @XmlElement(name="asrServer")
  public List<AsrServer> getItems() {
     return this.items;
  }

  public void setItems(List<AsrServer> items) {
     this.items = items;
  }


}
